src/styles/card.scss

```
@@ -1,18 +1,32 @@
+$color: #00acdc;
+
 .card {
   width: 200px;
 
   margin-bottom: 50px;
 
   border: 1px solid #f3f3f3;
   border-radius: 5px;
+
+  &__position {
+    margin: 0;
+    padding: 15px 14px;
+  }
+
   &__img {
     display: block;
+    width: 170px;
+    height: 164px;
+    margin-bottom: 15px;
+    padding: 12px 2px;
 
-    width: 160px;
-    height: 134px;
-    margin: 0 auto;
-    margin-top: 30px;
-    margin-bottom: 45px;
+    border: 3px solid white;
+    transition: transform 1s, border 1s ease;
```

```suggestion
    transition: transform 1s;
```
Извини что сразу не написал, но я только сейчас обратил внимание. Бордер должен сразу появляться, без задержки https://prnt.sc/ueaieb